docs/tools/Ide/intellij.md

```diff
@@ -4,6 +4,38 @@ sidebar_position: 3
 
 # IntelliJ IDEA
 
-- IntelliJ KCL Extension: [https://github.com/kcl-lang/intellij-kcl](https://github.com/kcl-lang/intellij-kcl)
+We provide two IntelliJ KCL plugin
 
-![intellij](/img/docs/tools/Ide/intellij/overview.png)
```

Comment:
Please avoid deleting this image as it may be referenced by other documents and can be replaced with the latest IDE KCL code screenshot.